src/controllers/clientController.js

```diff
@@ -137,3 +137,25 @@ exports.getChefs = async (req, res) => {
       .json({ error: `Error while retrieving chefs: ${error.message}` });
   }
 };
+
+exports.getChefsDishes = async (req, res) => {
```

Comment:
this is not your task!!

Reply:
Thanks for the feedback, I added it because when I was working on the nodemailer thingy I needed this endpoint for testing purposes so I kept it cuz I think we need it.